tests/system/Cache/Handlers/BaseHandlerTest.php

```diff
@@ -0,0 +1,66 @@
+<?php namespace CodeIgniter\Cache\Handlers;
+
+use stdClass;
+use CodeIgniter\Test\CIUnitTestCase;
+use Tests\Support\Cache\RestrictiveHandler;
+
+class BaseHandlerTest extends CIUnitTestCase
+{
+	/**
+	 * @dataProvider invalidTypeProvider
+	 */
+	public function testValidateKeyInvalidType($input)
+	{
+		$this->expectException('InvalidArgumentException');
+		$this->expectExceptionMessage('Cache key must be a string');
+
+		BaseHandler::validateKey(false);
```

Comment:
```suggestion
		BaseHandler::validateKey($input);
```

Reply:
🙄 Thanks Paul. I woodwork as well and have a rule that I never use saws when I'm tired - looks like Is would adopt the same for coding 😂